src/linagora.esn.calendar/app/calendar-configuration/calendar-configuration-tab-main/calendar-configuration-tab-main.controller.js

```diff
@@ -111,5 +112,13 @@ require('../../components/modals/calendar-delete-confirmation/calendar-delete-co
         })
         .catch(angular.noop);
     }
+
+    function exportCalendar() {
+      calendarAPI.exportCalendar(self.calendarIcsUrl, ({ data }) => {
+        const calendarBlob = new Blob([data], { type: CAL_EXPORT_FILE_TYPE });
+
+        FileSaver.saveAs(calendarBlob, self.calendarIcsExportedFileName);
+      });
+    }
```

Comment:
And this part can be extracted into a method in the calendar service (`calendarService` or something iirc).